docs/proposals/2019-02-14_Dropcap.md

```diff
@@ -0,0 +1,44 @@
+# Dropcap for paragraph and heading elements
+
+# Problem
+
+Users want the ability to add a dropcap to paragraphs and headings.
```

Comment:
What about other textual elements, such as: `blockquote`, `correction`, `interstitial_link`.

Reply:
For now, we are just focused on do ing text and header. It's possible those will need to be considered as well.

Reply:
What about people who want sub or sup scripts? I actually heard someone asking for being able to sup or sup words in the header as well. Would they be able to apply any other additional styles to the dropcap?

So I guess my only concern with this is that if we break this off into it's own **format** section then it might be confusing on what things in the format applies to, especially if we start adding things like color/size/style.

For instance, if we ended up adding format: { style: bold } would that apply to the entire header or just to the dropcap? It might make more sense to put this at the same level that we have the header:

```
content_elements: [{
  additional_properties: {},
  alignment: "left",
  channels: ["basic", "print"],
  content: "bla bla bla",
  level: 1,
  type: "header",
  dropcap: "letter",
  _id: "KKAE5ETLOVBLXCU7ZI4ELLVUJM"
}]
```